recipe/meta.yaml

```diff
@@ -57,23 +64,23 @@ requirements:
     - {{ cdt('libxext') }}               # [linux]
     - {{ cdt('libxext-devel') }}         # [linux]
     - {{ cdt('libxrender-devel') }}      # [linux]
-    - {{ cdt('mesa-libgl-devel') }}      # [linux and not s390x]
-    - {{ cdt('mesa-libegl') }}           # [linux and not (ppc64le or s390x)]
-    - {{ cdt('mesa-libgbm') }}           # [linux and not (ppc64le or s390x)]
-    - {{ cdt('mesa-libegl-devel') }}     # [linux and not (ppc64le or s390x)]
+    - {{ cdt('mesa-libgl-devel') }}      # [linux]
+    - {{ cdt('mesa-libegl') }}           # [linux]
+    - {{ cdt('mesa-libgbm') }}           # [linux]
+    - {{ cdt('mesa-libegl-devel') }}     # [linux]
     - {{ cdt('mesa-dri-drivers') }}      # [linux]
     - {{ cdt('libxau-devel') }}          # [linux]
-    - {{ cdt('alsa-lib-devel') }}        # [linux and not (ppc64le or s390x)]
+    - {{ cdt('alsa-lib-devel') }}        # [linux]
     - {{ cdt('gtk2-devel') }}            # [linux]
```

Comment:
Might be a concern since we're using GTK3 as dependencies later on (packaged by us). We should check library compatibility and if possible update to gkt3-devel